components/chart/index.tsx

```diff
@@ -171,14 +171,15 @@ function Chart({ data }: { config: Config; data: GraphContent[] }) {
       focusModule(nodeIdFromUrl.toString());
     }
   }, [focusModule]);
-
+  console.log(selectedModule);
   return (
     //@ts-ignore
     <HotKeys keyMap={SHORTCUT_KEYS} handlers={shortcutHandlers}>
       <div className="chart" id="chart">
         <Header
           showModulesSearch={setShowSearch}
           chartName={rootContent.title}
+          predefinedTags={[]}
```

Comment:
I think we should initialize the Header with selected tags coming from the URL query param